src/sentry/api/helpers/group_index/update.py

```diff
@@ -738,3 +715,45 @@ def update_groups(
         result["inbox"] = inbox
 
     return Response(result)
+
+
+def handle_is_public(
+    is_public: Any,
+    group_list: List[Group],
+    project_lookup: Dict[int, Project],
```

Comment:
```suggestion
    group_list: list[Group],
    project_lookup: dict[int, Project],
```

See https://typing.readthedocs.io/en/latest/source/best_practices.html#built-in-generics.